test/input/index.js

```diff
@@ -0,0 +1,60 @@
+const React = require('react');
+const { assert } = require('chai');
+const { mount } = require('enzyme');
+
+
+const Input = require('../../lib/input').default;
```

Comment:
test base input component too? there are some methods to test:
`getDefaultState`
`getValue`
`getCaretPosition`
`updateValue` 
check that `readDomValue` is called etc.